api/rbdigital.py

```diff
@@ -1002,7 +1102,7 @@ def validate_response(self, response, message, action=""):
         if response.status_code not in [200, 201]:
             if not message:
                 message = response.text
-            self.log.warning("%s call failed: %s ", action, message)
+            self.log.info("%s call failed: %s ", action, message)
```

Comment:
This is no longer a 'warning' scenario (if it ever was). It's very common that this call fails because we're looking up the barcode of a patron who doesn't have an account yet.